### ISSUE TITLE
ResNeXt (2017, Facebook AI Research)

### ISSUE THREAD

**Issue by taki0112** [zalandoresearch/fashion-mnist#58]
HI
When I tested the fashion mnist dataset with 50 epochs, the accuracy was up to 90%. Maybe it will be higher if you increase the epoch and adjust the hyperparameter.

Data augmentation has not been done.

https://github.com/taki0112/ResNeXt-Tensorflow

thank you

**Comment by kashif** [comment]
@taki0112 I suspect something is wrong with your ResNeXt implementation, since a simple 3 layer ConvNet will give about 91% accuracy after 10 or 12 epochs... So therefore kindly try to debug since I suspect ResNeXt should give you much more...

**Comment by taki0112** [comment]
Yes, thank you for your advice.
The residual layer was incorrectly coded and corrected. Let's try again.
